.eslintrc.cjs

```diff
@@ -136,7 +136,14 @@ module.exports = {
                 'plugin:@typescript-eslint/recommended',
             ],
             rules: {
-                '@stylistic/max-len': ['warn', 80],
+                '@stylistic/max-len': ['warn', {
+                    code: 100,
+                    comments: 80,
```

Comment:
Why not setting comments to 100 too ?

Reply:
I would have likely have 80 characters limit everywhere (which is pretty standard in C-like languages). However we have long functions and methods name in long chain which in practice eat the 80 characters limit... So I decided to change it only for code, I could put comment too in 100 characters if you wish !

Reply:
I agree that 80 is more common. Generally it's either 80 or 120 but 100 is fine by me too. However I think it would be easier to have the same limit for both comments and code.